diff --git a/libujson-only.json b/libujson-only.json
new file mode 100644
index 0000000..7fcb2e3
--- /dev/null
+++ b/libujson-only.json
@@ -0,0 +1 @@
+["libujson"]
diff --git a/parsers/test_libujson.sh b/parsers/test_libujson.sh
new file mode 100755
index 0000000..3889c38
--- /dev/null
+++ b/parsers/test_libujson.sh
@@ -0,0 +1,2 @@
+#!/bin/sh
+../../ujson-parse-test $*
diff --git a/run_tests.py b/run_tests.py
index 462edb6..efae0d1 100755
--- a/run_tests.py
+++ b/run_tests.py
@@ -41,6 +41,11 @@ programs = {
     #         "url":"https://github.com/step-/JSON.awk",
     #         "commands":["/usr/bin/mawk", "-f", os.path.join(PARSERS_DIR, "test_JSON.awk", "callbacks.awk"), "-f", os.path.join(PARSERS_DIR, "test_JSON.awk", "JSON.awk")]
     #     },
+    "libujson":
+        {
+            "url":"",
+            "commands":[os.path.join(PARSERS_DIR, "test_libujson.sh")],
+        },
     "Bash JSON.sh 2016-08-12":
         {
             "url":"https://github.com/dominictarr/JSON.sh",
@@ -935,8 +940,8 @@ def generate_report(report_path, keep_only_first_result_in_set = False):
 
         </HTML>
         """)
-    if os.path.exists('/usr/bin/open'):
-        os.system('/usr/bin/open "%s"' % report_path)
+    #if os.path.exists('/usr/bin/open'):
+    #    os.system('/usr/bin/open "%s"' % report_path)
 
 ###
 
